decentraland/unity-renderer
# Chore: applied suggested graphical changes to channels

## What does this PR change?

Fix #2913
Applies all changes listed in the issue
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/notification-improvs&ENABLE_NOTIFICATION_CHAT=
2. Create new messages with keys "k" and "g"
3. Verify that all the listed improvements in the issue work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
